data/json/items/vehicle/baskets.json

```diff
@@ -0,0 +1,37 @@
+[
```

Comment:
Can this file be merged with cargo.json? Seems like an arbitrary distinction

Reply:
Yeah, that was an artifact of the script I used to set up the copy-from.  I'll fix that.